### ISSUE TITLE
Cannot publish fork

### ISSUE THREAD

**Issue by subjective** [peer/doc#166]
I'm using the merge/fork test version for Unifesp.  When I fork a document and try to publish the fork, I get an error: "An error has occurred when trying to publish the document."  Is this the expected behavior?

**Comment by mitar** [comment]
Where do you get this error. From PeerDoc or from Appcivist? You should always try to do things through AppCivist. In that case PeerDoc should not show you any error.

But yes, publishing a fork should be possible.

**Comment by subjective** [comment]
I'm using the version that is embedded on AppCivist.  But I was using the buttons on the PeerDoc embed.  When I try the buttons on AppCivist, I get a different problem: when I try to fork a document, I get an error on AppCivist: "Error trying to fork. Please try again later."  The console reports:

```
XML Parsing Error: no root element found
Location: https://testplatform.appcivist.org/api/log/front
Line Number 1, Column 1:
```

**Comment by mitar** [comment]
Please report that to AppCivist. :-)

Maye you can run PeerDoc locally on your machine and see which of things you are observing are because of PeerDoc and which one are because of AppCivist.

**Comment by mitar** [comment]
But I think this will be fixed by some fixes inside #154.

**Comment by mitar** [comment]
I think this has been now resolved. Thanks for reporting.
